content/applications/sales/sales/products_prices/products/variants.rst

```diff
@@ -2,180 +2,278 @@
 
-.. note::
+.. important::
    Once added to a product, an attribute's :guilabel:`Variants Creation Mode` cannot be edited.
 
-Values should be added to an attribute before saving, but more values can be added at any time,
-if needed. To add a value, click :guilabel:`Add a line`. From there, you can:
+Lastly, the :guilabel:`eCommerce Filter Visibility` field determines whether these attribute options
+are visible to the customer on the front-end, as they shop on the online store.
 
-#. Type in the value's name.
-#. Check a box to indicate whether or not the value is custom (i.e. the customer provides unique
-   specifications).
-#. Specifically for colors, add an HTML color code to make it even easier for salespeople and
-   customers to know what they're selecting.
+- :guilabel:`Visible`: the attribute values are visible to customers on the front-end.
+- :guilabel:`Hidden`: the attribute values are hidden from customers on the front-end.
 
-A color code can be selected either by dragging the slider around or by entering a specific HTML
-color code (e.g. #FF0077).
+Attribute values
+----------------
 
-.. image:: variants/picking-a-color.png
-   :align: center
-   :alt: Selecting a color.
+Attribute values should be added to the :guilabel:`Attribute Values` tab. Values can be added to an
+attribute at any time, if needed.
+
+To add a value, click :guilabel:`Add a line` in the :guilabel:`Attribute Values` tab.
+
+Then, enter the name of the value in the :guilabel:`Value` column. Next, check the box in the
+:guilabel:`Is custom value` column, if the value is custom (i.e. the customer gets to provide unique
+specifications that are specific to this particular value).
+
+.. note::
+   If the :guilabel:`Display Type` - :guilabel:`Color` option is selected, the option to add an HTML
+   color code to the far-right of the value line, to make it easier for salespeople and customers to
+   know exactly what color option they're choosing.
+
+   .. image:: variants/attribute-value-add-a-line.png
+      :align: center
+      :alt: Attribute values tab when add a line is clicked, showing the custom columns.
+
+   To choose a color, click the blank circle in the :guilabel:`Color` column, which reveals a HTML
+   color selector pop-up window.
 
-Finally, click :guilabel:`Save` to finish creating the attribute.
+   .. image:: variants/picking-a-color.png
+      :align: center
+      :alt: Selecting a color from the HTML color pop-up window that appears on attribute form.
+
+   In this pop-up window, select a specific color by dragging the color slider to a particular hue,
+   and clicking on the color portion directly on the color gradient window.
+
+   Or, choose a specific color by clicking the *dropper* icon, and selecting a desired color that's
+   currently clickable on the screen.
+
+   A color can also be chosen by entering a specific HTML color code into the corresponding
+   :guilabel:`R`, :guilabel:`G`, and :guilabel:`B` fields.
```

Comment:
Can also use HEX or HSL values by clicking through the arrows to change up the display